templates/snmpd/snmpd.conf.erb

```diff
@@ -5,23 +5,23 @@
 ################################################################################
```

Comment:
Why do we have a mix of erb and epp files?  Seems like we should be one or the other.